packages/varnish/src/theme/theme.ts

```diff
@@ -60,63 +83,120 @@ export const getVarnishDesignTokens = (): ThemeOptions => {
           }),
         },
       },
-      MuiAppBar: {
```

Comment:
rationale behind adding and then removing these?

Reply:
turned out that the override was not needed...
as i suspected, the base theme changes trickled down and made this fix for us...